integration/tests/crud/creategetdelete_test.go

```diff
@@ -428,6 +415,50 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
+		When("scaling nodegroup(s)", func() {
+			It("should scale a single nodegroup", func() {
+				By("passing the name of the nodegroup as a flag")
+				cmd := params.EksctlScaleNodeGroupCmd.WithArgs(
+					"--cluster", params.ClusterName,
+					"--nodes-min", "3",
+					"--nodes", "4",
+					"--nodes-max", "5",
+					"--name", initNG,
+				)
+				Expect(cmd).To(RunSuccessfully())
+			})
+
+			It("should scale all nodegroups", func() {
+				By("scaling all nodegroups in the config file o the desired capacity, max size, and min size")
+				cmd := params.EksctlScaleNodeGroupCmd.WithArgs(
+					"--config-file", "-",
+				).
+					WithoutArg("--region", params.Region).
+					WithStdin(clusterutils.ReaderFromFile(params.ClusterName, params.Region, "testdata/scale-nodegroups.yaml"))
```

Comment:
I have mixed feelings about this :/ I really dislike that I have to go look elsewhere to see whats in this file, vs in-lining the config like using the struct and then writing it too a file

Reply:
![image](https://user-images.githubusercontent.com/18622989/142210042-7d409e53-0284-40f9-81dc-e823b7e1021b.png)

Reply:
hehehehe

Reply:
![image](https://user-images.githubusercontent.com/19669095/142244139-1ae2dafa-a13a-4632-a611-5e9408003120.png)

Reply:
AMAZING